make_BAF_igv.py

```diff
@@ -0,0 +1,60 @@
+#! /usr/bin/env python3
+import sys
+import argparse
+import vcf
+
+def calc_baf(args):
+    output_file = open(args.outputfile, 'w')
```

Comment:
should you check if that outputfile already exists before overwriting it?

Reply:
I think that only applies if you want to keep/protect the original outputfile.
I would argue that overwriting should be default as re-analysis will require this in most cases (without manual interventions like removing a file).

Reply:
I did change required output:  output (-o) is optional (and does overwrite existing outputfile). 
Default is now print to stdout.